doc/reference/pipeline_spec.md

```diff
@@ -619,6 +617,31 @@ The root mount point is at `/pfs`, which contains:
   name if not specified.
 - `/pfs/out` which is where you write any output.
 
-### Output Formats
-
-PFS supports data to be delimited by line, JSON, or binary blobs.
+# Environment Variables
+
+There are several environment variables that get injected into the user code
+before it runs. They are:
+
+- `PACH_JOB_ID` the id the currently run job.
+- `PACH_OUTPUT_COMMIT_ID` the id of the commit being outputted to.
```

Comment:
Had a user ask if input commit could also be an Env Var. Of course, he know he can get it from the output commit + prov, but thought having it directly would be more convenient.